backend/src/appointment/routes/schedule.py

```diff
@@ -73,10 +78,9 @@ def is_this_a_valid_booking_time(schedule: models.Schedule, booking_slot: schema
 
     booking_slot_end = booking_slot.start + timedelta(minutes=schedule.slot_duration)
 
-    if (
-        booking_slot.start < datetime.combine(today, schedule.start_time, tzinfo=timezone.utc)
-        or booking_slot_end > datetime.combine(today, schedule.end_time, tzinfo=timezone.utc) + timedelta(days=add_day)
-    ):
+    if booking_slot.start < datetime.combine(
+        today, schedule.start_time, tzinfo=timezone.utc
+    ) or booking_slot_end > datetime.combine(today, schedule.end_time, tzinfo=timezone.utc) + timedelta(days=add_day):
```

Comment:
Can we move this into separate vars to make it a little bit more readable? E.g.:

```suggestion
    too_early = booking_slot.start < datetime.combine(today, schedule.start_time, tzinfo=timezone.utc)
    too_late = booking_slot_end > (
      datetime.combine(today, schedule.end_time, tzinfo=timezone.utc) + timedelta(days=add_day)
    )
    if too_early or too_late:
```